### PR TITLE
Better UI for patient sign-up and reset-password pages

### PR DESCRIPTION
**Pull request by NBallaney**
Update how we display the password requirements to make them easier to understand

Related to https://github.com/IoraHealth/snowflake/pull/612
Trello: https://trello.com/c/fiPz724T/290-friendlier-ui-ux-for-displaying-password-requirements-on-reset-password-and-sign-up-pages

### CODE REVIEW

**Comment by NBallaney** [comment]
[(2) Test for invalid and error cases like: "fails for invalid medication hashes" and "either succeeds for all or fails for all i.e. it is a Transaction" (needs to be done)](https://trello.com/c/xxIfeepu/349-2-test-for-invalid-and-error-cases-like-fails-for-invalid-medication-hashes-and-either-succeeds-for-all-or-fails-for-all-ie-it-i)

**Comment by NBallaney** [comment]
[Friendlier UI & UX for displaying password requirements on reset password and sign up pages](https://trello.com/c/fiPz724T/290-friendlier-ui-ux-for-displaying-password-requirements-on-reset-password-and-sign-up-pages)

File: dist/jquery-password-validator.css [hunk 1]
```diff
@@ -1,38 +1,37 @@
 .jq-password-validator__popover {
-  background: white;
+  background: #fff;
```
Comment (SViccari) [inline dist/jquery-password-validator.css:2]: Just curious, how many applications use this plugin? Is it just the Patient app?

File: dist/jquery-password-validator.css [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
-    transform: scale(1);
-    visibility: visible; }
+  .jq-password-validator__rule.is-valid {
+    color: green; }
+    .jq-password-validator__rule.is-valid .jq-password-validator__checkmark {
+      transform: scale(1);
+      visibility: visible;
+      fill: green; }
+
+/*# sourceMappingURL=jquery-password-validator.css.map */
```
Comment (SViccari) [inline dist/jquery-password-validator.css:37]: Can this comment be removed? ✂️

File: dist/jquery.password-validator.js [hunk 3]
```diff
@@ -55,160 +55,154 @@ __p += '<div class="jq-password-validator__rule ' +
...
-		$.fn[ pluginName ] = function ( options ) {
-				return this.each(function() {
-						if ( !$.data( this, "plugin_" + pluginName ) ) {
-								$.data( this, "plugin_" + pluginName, new Plugin( this, options ) );
-						}
-				});
-		};
-
-})( jQuery, window, document );
+(function ($, window, document, undefined) {
```
Comment (SViccari) [inline dist/jquery.password-validator.js:58]: What prompted the changes to this file? Are the changes mostly formatting changes?

Comment (NBallaney) [reply dist/jquery.password-validator.js:58]: Yes, mostly prettier. Sorry, I didn't think of putting the formatting changes in a separate commit at the time

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jquery-password-validator",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (SViccari) [inline package.json:3]: The current version bump indicates we made a backward-compatible bug fix. You'll know best but I'm wondering if this should be a minor or major version bump instead.

File: dist/jquery-password-validator.css (hunk 1 above)
Comment (NBallaney) [reply dist/jquery-password-validator.css:2]: As far as I know, only Snowflake uses it for the patient app.

File: package.json (hunk 4 above)
Comment (alexrothenberg) [reply package.json:3]: :trollface: before v1.0 semver has no guarantees 
image

File: dist/jquery-password-validator.css (hunk 2 above)
Comment (NBallaney) [reply dist/jquery-password-validator.css:37]: It was there from before. I don't fully understand it, so I let it be

Comment (SViccari) [reply dist/jquery-password-validator.css:37]: I don't see it in the [main branch](https://github.com/IoraHealth/jquery-password-validator/blob/master/dist/jquery-password-validator.css). Perhaps I'm overlooking it?

File: package.json (hunk 4 above)
Comment (SViccari) [reply package.json:3]: Lol, I forgot about that, good point. 👍 

chaos

File: dist/jquery-password-validator.css (hunk 2 above)
Comment (NBallaney) [reply dist/jquery-password-validator.css:37]: Good call. I just saw, it's not there. All the files in this folder are created by the `grunt` task. So I'm betting this was put by that task.

**Pull request merged**